apps/src/templates/sectionProgress/SectionProgress.jsx

```diff
@@ -82,6 +84,23 @@ class SectionProgress extends Component {
     this.props.loadScript(this.props.scriptId);
   }
 
+  componentDidUpdate(prevProps) {
+    // Check if we are going from a CSF script to a non-CSF script and
+    // currentView is Standards. If so re-set currentView to Summary since
+    // Standards doesn't apply.
+    const isCSFBefore =
+      prevProps.scriptData && !prevProps.scriptData.excludeCsfColumnInLegend;
```

Comment:
Should we rename `excludeCsfColumnInLegend` since we are using it for more than just that?

Reply:
This is a pull-the-string-unravel-the-sweater-situation because I think we can actually eliminate `excludeCsfColumnInLegend` as a script property in favor of `curriculum_umbrella == CSF`.  That seems worth doing and timely but also big enough to be its own work item, so I'll tackle that then circle back to make complementary changes here.

Reply:
Thanks for tackling that @Erin007 !

Reply:
https://github.com/code-dot-org/code-dot-org/pull/32470 is out for review